src/vorta/scheduler.py

```diff
@@ -333,24 +333,26 @@ def reload_all_timers(self):
             self.set_timer_for_profile(profile.id)
 
     def next_job(self):
-        next_job = now = dt.now()
-        next_profile = None
-        for profile_id, timer in self.timers.items():
-            if timer['type'] != ScheduleStatusType.SCHEDULED:
-                continue
-
-            if next_job == now and timer['dt'] > next_job and timer['qtt'].isActive():
-                next_job = timer['dt']
-                next_profile = profile_id
-            elif next_job > now and timer['dt'] < next_job and timer['qtt'].isActive():
-                next_job = timer['dt']
-                next_profile = profile_id
-
-        if next_profile is not None:
-            profile = BackupProfileModel.get_or_none(id=next_profile)
-            return f"{next_job.strftime('%H:%M')} ({profile.name})"
-        else:
-            return self.tr('None scheduled')
+        now = dt.now()
+
+        def is_scheduled(timer):
+            return (
+                timer["type"] == ScheduleStatusType.SCHEDULED
+                and timer["qtt"].isActive()
+                and timer["dt"] >= now
+            )
+
+        next_job = min([(profile_id, timer['dt']) for profile_id, timer in self.timers.items() if is_scheduled(timer)], key=lambda i: i[0], default=None)
+        if not next_job:
+            return self.tr("None scheduled")
+        profile_id, time = next_job
+        profile = BackupProfileModel.get_or_none(id=profile_id)
+
+        time_format = "%H:%M"
+        if time - now > timedelta(days=1):
+            time_format = "%d-%b %H:%M"
```

Comment:
This results in a string like `01-Jun 14:43`.  Instead you could use [QLocale.toString](https://doc.qt.io/qt-5/qlocale.html#toString-19) to format the string in a localized manner. This is already done here:
https://github.com/borgbase/vorta/blob/e58c83b5c9e2b57d31eaed3f3b8edeeceef852ef/src/vorta/views/schedule_tab.py#L162-L164

The  `NarrowFormat` might be better suited though.

Reply:
added